guides/nodejs.md

```diff
@@ -0,0 +1,308 @@
+# Node.js Developer's Guide
+
+When creating a Node.js function using the `func` binary, the project directory
+looks like a typical Node.js project. Both HTTP and Event functions have the same
+template structure.
+
+```
+❯ tree
+.
+├── func.yaml
+├── index.js
+├── package.json
+├── README.md
+└── test
+    ├── integration.js
+    └── unit.js
+```
+
+Aside from the `func.yaml` file, this looks like the beginning of just about
+any Node.js project. For now, we will ignore the `func.yaml` file, and just
+say that it is a configuration file that is used when building your project.
+If you're really interested, check out the [reference doc](config-reference.doc).
+
+## Running the function locally
+
+To run a function, you'll first need to build it. This step creates an OCI
+container image that can be run locally on your computer, or on a Kubernetes
+cluster.
+
+```
+❯ func build
+```
+
+After the function has been built, it can be run locally.
+
+```
+❯ func run
+```
+
+Functions can be invoked with a simple HTTP request. 
+You can test to see if the function is working by using your browser to visit
+http://localhost:8080. You can also access liveness and readiness
+endpoints at http://localhost:8080/health/liveness and
+http://localhost:8080/health/readiness. These two endpoints are used
+by Kubernetes to determine the health of your function. If everything
+is good, both of these will return `OK`.
```

Comment:
Mayb add a short paragraph on running the function on cluster (with the command).